src/frontend/planner_test/tests/testdata/output/nexmark.yaml

```diff
@@ -1515,12 +1515,13 @@
     │     │ └─BatchScan { table: auction, columns: [auction.id, auction.item_name], distribution: UpstreamHashShard(auction.id) }
     │     └─BatchExchange { order: [], dist: HashShard(bid.auction) }
     │       └─BatchScan { table: bid, columns: [bid.auction], distribution: SomeShard }
-    └─BatchProject { exprs: [(sum0(count) / count(bid.auction)) as $expr1] }
-      └─BatchSimpleAgg { aggs: [sum0(count), count(bid.auction)] }
+    └─BatchProject { exprs: [(sum0(sum0(count)) / sum0(count(bid.auction))) as $expr1] }
+      └─BatchSimpleAgg { aggs: [sum0(sum0(count)), sum0(count(bid.auction))] }
```

Comment:
Looks like the nested 2-phase is because `DistinctAggRule` first 2-phased the call before `to_stream` generating 2-phase plan. The plan seems correct to me.

Reply:
I see. 👍